features/LAY000_Layers-information.feature

```diff
@@ -0,0 +1,19 @@
+@implementer-agreement
+@LAY
+@version1
+@E00020
+
+Feature: LAY000 - Layers Information
+    The rule verifies the presence of IFC entities used to assign layers (also known as, CAD layer) to collection of elements. 
+    This is used mainly for grouping and visibility control, and in general to organise geometry into groups that may be shown or hidden.
+    https://ifc43-docs.standards.buildingsmart.org/IFC/RELEASE/IFC4x3/HTML/concepts/Product_Shape/Product_Geometry_Layer/content.html
+
+
+    Scenario: Check for activation
+
+    Given an IfcProduct
+    Given its attribute Representation
+    Given its attribute Representations
+    Given its attribute LayerAssignments
```

Comment:
Is the check that this is not empty in the `The IFC model contains information on` implementation? That's a bit implicit.

Should there not be something like:

   Given its attribute LayerAssignments is not empty

or separated into two steps?

Reply:
The two statements are similar and, for activation, they function the same. The difference lies in which entity is selected for the next step. For these 000 rules, we don't evaluate the instances but simply check if they exist in the model. If the `LayerAssignments` attribute in an IFC file isn't empty, the rule will activate regardless of which statement we use.

Currently, the last instances in the stack (if any) are of type IfcPresentationLayerAssignment. With the 'is not empty' statement, they will be of type IfcShapeRepresentation (filtered out by the current statement from the previous Given statement).